src/main.tsx

```diff
@@ -12,7 +12,20 @@ const client = new ApolloClient({
       ? //created environment variables must be prefixed by VITE
         "https://synthesis-workshop-backend-97f537f332bd.herokuapp.com/api/graphql"
       : "http://localhost:8080/api/graphql",
-  cache: new InMemoryCache(),
+  cache: new InMemoryCache({
```

Comment:
In order to do pagination queries where we load more results, we need to tell the in memory cache how to merge new query results with old ones. The `keyArgs` array below tells the query when it should and should not invalidate the query. In our case, the `skip` keyArg tells the cache it should merge results of the new query with the old ones, but if it receives a query with a different `orderBy` or `take` keyArg, it should invalidate the query and replace the old results with the new one.

Feel free to read more here:
https://www.apollographql.com/docs/react/pagination/core-api/#merging-paginated-results